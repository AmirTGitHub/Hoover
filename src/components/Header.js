import React from 'react';

const Header = (props) =>{
    return(
        <div className='header'>
            <h1 className='header--title' >{props.headerTitle}</h1>
        </div>
    )
}

export default Header