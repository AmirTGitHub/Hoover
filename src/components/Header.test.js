import Header from './Header'

it('check heather render corectly', () => {
    const wrapper = render(
        <Header title='hoover app' />
    )
    expect(wrapper).toMatchSnapshot()
})