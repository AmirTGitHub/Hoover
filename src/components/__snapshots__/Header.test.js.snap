// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check heather render corectly 1`] = `
<div
  class="header"
>
  <h1
    class="header--title"
  >
    
  </h1>
</div>
`;
